chrome_manifest_v3/lib/combo.js

```
@@ -1,63 +1,58 @@
+          },
+        })
+        .keyup(event => {
           if (event.which == 13) {
-            var forAttr = $(event.target).parent().attr("for-select");
+            var forAttr = $(event.target).parent().attr('for-select');
             if (forAttr) {
-              var nextSelAttr = $("#" + forAttr).attr("next-select");
-              var nextSel = $("#" + nextSelAttr);
-              if ($(nextSel).hasClass("g2t-custom-combobox")) {
-                $(nextSel).find("input").focus();
+              var nextSelAttr = $('#' + forAttr).attr('next-select');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)